tests/integration/test_gcs_filesystem.py

```diff
@@ -0,0 +1,35 @@
+import uuid
+
+from airflow.hooks.base import BaseHook
+
+from airflow_tools.filesystems.filesystem_factory import FilesystemFactory
+from airflow_tools.filesystems.impl.google_cloud_storage_filesystem import GCSFilesystem
+
+
+def test_gcs_filesystem():
+    """Expects an environment variable set to a GCS test bucket.
+    AIRFLOW_CONN_GCP_DATA_LAKE_TEST='{"conn_type": "google_cloud_platform", "extra": {"key_path": "/.../keyfile.json"}}'"""
```

Comment:
I assume we have an static bucket for this test in GCS, right?

Reply:
I have used a static bucket that we have but I think it's not in Pulumi. We should probably create an official one.